### PR TITLE
Interactions

### PR DESCRIPTION
**Pull request by kendevops**
_Description of changes:_

Made changes on on the interactions snippets and updated the react and react native chat bot prefix so that they are different from each other.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: vscode/snippets/interactions.code-snippets [hunk 1]
```diff
@@ -1,84 +1,85 @@
...
-        "prefix": "handleComplete",
-        "scope": "javascript,javascriptreact,typescript,typescriptreact",
-        "body": [
-            "handleComplete(err, confirmation) {",
+                "\t})}",
+			"/>"
+		]
+	},
+	"handleComplete React Native": {
+		"prefix": "handleComplete",
```
Comment (wlee221) [inline vscode/snippets/interactions.code-snippets:72]: maybe put `handleComplete native` like line 55.

File: vscode/snippets/interactions.code-snippets [hunk 2]
```diff
@@ -1,84 +1,85 @@
 {
-    "Interactions Manual Configuration": {
-        "prefix": "Interactions",
-        "scope": "javascript,javascriptreact,typescript,typescriptreact",
-        "body": [
-            "import Amplify from 'aws-amplify';\n",
-            "Amplify.configure({",
+	"Interactions Manual Configuration": {
+		"prefix": "Interactions",
```
Comment (wlee221) [inline vscode/snippets/interactions.code-snippets:3]: Maybe we should do `Interactions.config` to be consistent with Predictions.

File: vscode/snippets/interactions.code-snippets (hunk 1 above)
Comment (wlee221) [inline vscode/snippets/interactions.code-snippets:72]: `handleComplete Native` on line 72 and `handleComplete React` on line 25 for separation

File: vscode/snippets/interactions.code-snippets (hunk 2 above)
Comment (kendevops) [reply vscode/snippets/interactions.code-snippets:3]: Yeah i thought of that and wanted a confirmation to implement that.

**Pull request closed without merging**